src/Directory.Build.targets

```diff
@@ -4,5 +4,8 @@
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\opensource.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
+  <ItemGroup>
+    <None Remove="**/bin/**;**/obj/**;**/*.meta" />
```

Comment:
Can you add an `<!-- -->` comment explaining what you explained in the PR description?

Reply:
ok